uaaextras/templates/email/body.html

```diff
@@ -17,18 +17,18 @@
         <p>
         <ol>
             <li>
-            <b>Accept the invitation</b> - <a href="{{verification_url}}">Accept your invite</a> to continue the registration process. (The invitation will expire after 24 hours.) You can also copy the URL below and paste it into your browser's address bar:
+            <b>Accept the invitation</b> - <a href="{{verification_url}}">Accept your invite [2]</a> to continue the registration process. (The invitation will expire after 24 hours.) You can also copy the URL below and paste it into your browser's address bar:
             <br />
             {{verification_url}}
             </li>
             <li>
-	          <b>Read the documentation</b> - After you register and log in, review the <a href="https://cloud.gov/docs/getting-started/accounts/#use-your-account-responsibly">acceptable uses and rules of behavior</a>.
+              <b>Read the documentation</b> - After you register and log in, review the <a href="https://cloud.gov/docs/getting-started/accounts/#use-your-account-responsibly">acceptable uses and rules of behavior [3]</a>.
             </li>
             <li>
-            If you signed up for a trial space (a sandbox), <strong>make sure you understand the <a href="https://cloud.gov/overview/pricing/free-limited-sandbox/#to-keep-in-mind-before-you-try-your-sandbox"><strong>sandbox policies and restrictions</a></strong>.
+            If you signed up for a trial space (a sandbox), <strong>make sure you understand the <a href="https://cloud.gov/overview/pricing/free-limited-sandbox/#to-keep-in-mind-before-you-try-your-sandbox">sandbox policies and restrictions [4]</a></strong>.
             </li>
             <li>
-            Then <a href="https://cloud.gov/docs/getting-started/setup/">set up your access and get started</a>.
+            Then <a href="https://cloud.gov/docs/getting-started/setup/">set up your access and get started [5]</a>.
```

Comment:
Might be nice to put this line's call to action in boldface too, for consistency with the other lines.

Reply:
Done!